app/src/pages/SignUp.tsx

```diff
@@ -1,234 +1,272 @@
-import * as React from "react";
-import { useNavigate } from "react-router-dom";
-import { LoadingButton } from "@mui/lab";
-import Avatar from "@mui/material/Avatar";
-import CssBaseline from "@mui/material/CssBaseline";
-import Grid from "@mui/material/Grid";
-import Box from "@mui/material/Box";
-import LockOutlinedIcon from "@mui/icons-material/LockOutlined";
-import Typography from "@mui/material/Typography";
-import Container from "@mui/material/Container";
-import { createTheme, ThemeProvider } from "@mui/material/styles";
-import Stepper from "@mui/material/Stepper";
-import Step from "@mui/material/Step";
-import StepLabel from "@mui/material/StepLabel";
-import Button from "@mui/material/Button";
-
-import { MainClientContext } from "../service/AppService";
-import SignUpForm from "../modules/SignUp/Form";
-import ChooseAvatar from "../modules/SignUp/ChooseAvatar";
-import Copyright from "../components/Copyright";
-import { Pages } from "../types/enums";
-import { IUser } from "../types/models.types";
-
-const THEME = createTheme();
-const steps = ["Основные данные", "Выбор аватара"];
+import * as React from "react"
```

Comment:
кстати, здесь можно просто писать import React from "react".
При этом, прочитай пж (найди где нить инфу) о том, в какой момент это реально нужно использовать и отпиши пж мне, обсудим как раз

Reply:
As dor me i think nowadays we may not to import react as import React from "react". in react 18
correct me if i,m wrong


as for this import * as React from "react"
i think we shoud import this way? what do you think?
because it more optimize
![image](https://github.com/user-attachments/assets/a99e3e30-7265-467e-b296-a587a43b8694)